src/components/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
What's this file for? Common utility components?

Reply:
Yea, I'd probably just make a LinkButton component and then have a styles folder serperate from components for styles like ContentContainer.

Reply:
I can move LinkButton out into its own component folder